src/Pages/GlobalConfigurations/ClustersAndEnvironments/ClusterEnvironmentDrawer/types.ts

```diff
@@ -0,0 +1,41 @@
+import { TagType } from '@devtron-labs/devtron-fe-common-lib'
+
+export interface ClusterEnvironmentDrawerProps {
+    environmentName: string
+    namespace: string
+    id: string
+    clusterId: number
+    prometheusEndpoint: string
+    isProduction: boolean
+    description: string
+    reload: () => void
+    hideClusterDrawer: () => void
+    isVirtual: boolean
+}
+
+export interface ClusterEnvironmentDrawerFormProps {
```

Comment:
We should extend `ClusterEnvironmentDrawerProps` from `ClusterEnvironmentDrawerFormProps`